src/etl/disease_etl.py

```diff
@@ -298,8 +298,8 @@ def get_generators(self, disease_data, batch_size, data_provider):
             do_id = disease_record.get('DOid')
 
 
-            if 'qualifier' in disease_record:
-                qualifier = disease_record.get('qualifier')
+            if 'negation' in disease_record:
+                qualifier = disease_record.get('negation')
```

Comment:
Was this an actual change in the file? Or a mis applied search and replace?

Reply:
The file changed from 'qualifier' to 'negation' - part of a previous PR I think.